src/functionalTest/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsControllerTest.java

```diff
@@ -26,31 +38,146 @@
 
     @Test
     public void should_return_empty_list_of_jobs() throws Exception {
-        send("")
+        sendGetRequest()
             .andExpect(status().isOk())
-            .andExpect(content().json(
-                "{"
-                    + "\"content\":[],"
-                    + "\"total_pages\":0,"
-                    + "\"total_elements\":0,"
-                    + "\"last\":true,"
-                    + "\"size\":10,"
-                    + "\"number\":0,"
-                    + "\"sort\":null,"
-                    + "\"first\":true,"
-                    + "\"number_of_elements\":0"
-                    + "}"
-            ))
+            .andExpect(jsonPath("content").isEmpty())
+            .andExpect(jsonPath("total_pages").value(0))
+            .andExpect(jsonPath("total_elements").value(0))
+            .andExpect(jsonPath("last").value(true))
+            .andExpect(jsonPath("size").value(10))
+            .andExpect(jsonPath("number").value(0))
+            .andExpect(jsonPath("sort").value(isNull()))
+            .andExpect(jsonPath("first").value(true))
+            .andExpect(jsonPath("number_of_elements").value(0));
+    }
+
+    @Test
+    public void should_create_a_new_job_and_return_job_uri_of_newly_created_job_when_valid_job_input_is_passed()
+        throws Exception {
+        //Given
+        String jobJson = jobJson();
+
+        //when
+        MvcResult mvcResult = sendPostRequest(jobJson)
+            .andExpect(status().isCreated())
+            .andReturn();
+
+        //then
+        //Create a new URI object which will validate whether the url we got back is an uri or not.
+        URI newJobUri = new URI(mvcResult.getResponse().getRedirectedUrl());
```

Comment:
it's not really a redirect url. imho it would be cleare to use `.getHeader(LOCATION_HEADER)`